stacks/api/function/horizon/src/enqueuer.ts

```diff
@@ -1,7 +1,7 @@
 import {Enqueuer} from "@spica-server/function/enqueuer";
 import {EventQueue, Queue} from "@spica-server/function/queue";
 
-export const ENQUEUER = "HORIZON_SCHEDULER";
+export const ENQUEUER = Symbol.for("HORIZON_SCHEDULER");
```

Comment:
Why did we decide to use 'Symbol'?

Reply:
I wanted to prevent the misuse of injection tokens so I decided to use something like this.

Reply:
In the previous code, you could use both `@Inject(HORIZON)` and `@Inject("HORIZON_SCHEDULER")` since the string type is a value type so both of them would work if I left the code as it was.